meta-lmp-base/recipes-samples/images/lmp-gateway-image.bb

```diff
@@ -10,7 +10,6 @@ require lmp-feature-nat64.inc
 require lmp-feature-jobserv.inc
 require lmp-feature-debug.inc
 require lmp-feature-softhsm.inc
-require lmp-feature-sbin-path-helper.inc
```

Comment:
Do you also want to delete the `lmp-feature-sbin-path-helper.inc` file here as well?

Reply:
Didn't delete just yet as we might have factory users importing such .inc (it won't cause any bad side effect, it just won't be necessary anymore).

Was thinking on keeping it for a few more releases and then we can safely remove it, what do you think?

Reply:
Good point.